warn about camelCase mappings in registerPrimitive

morandd opened the pull request:
Added a warning in registerPrimitive when the user has provided mappings keys that are not all lower case. Since the DOM reports all element attributes in lower case (regardless of how the user typed them in the HTML source), the mappings keys must also be lower case.

This issue was discussed here
  https://github.com/aframevr/aframe/issues/2629
A separate PR was previously accepted to update the documentation to mention that mappings are case-sensitive.

**Description:**

**Changes proposed:**
-
-
-

ngokevin commented on src/extras/primitives/primitives.js:
```diff
@@ -45,6 +45,7 @@ module.exports.registerPrimitive = function registerPrimitive (name, definition)
           var self = this;
           Object.keys(mappings).forEach(function resolveCollision (key) {
             var newAttribute;
+            if (key != key.toLowerCase()) console.warn('Mapping keys should be specified in lower case. This mapping may not be recognized');
```
You can use `warn` https://github.com/morandd/aframe/blob/9e0f76f70927cbb9d1936f2dcd59f7d06654586d/src/extras/primitives/primitives.js#L9

ngokevin replied:
`!==`

ngokevin replied:
Use `braces` for `if` statements

morandd commented:
switched from console.warn to warn() and added braces around if. I hope it's OK (but please take a quick look - I'm not such a genius with javascript)

ngokevin commented:
Thanks. No problem, just style / minor stuff.

ngokevin commented:
Still need to adjust one more thing. Switch `!=` to `!==`. 

https://travis-ci.org/aframevr/aframe/jobs/225977249

ngokevin commented:
thanks!
